lib/src/main/java/org/sonarsource/scanner/lib/internal/facade/AbstractScannerEngineFacade.java

```diff
@@ -82,4 +83,17 @@ private static void initAnalysisProperties(Map<String, String> p) {
   public Map<String, String> getBootstrapProperties() {
     return bootstrapProperties;
   }
+
+  @Override
+  public String getServerType() {
```

Comment:
This is minor, but I tend to expect an `enum` when a method is named `getXXXType`. I would suggest changing it to:
```suggestion
  public String getServerLabel() {
```
or something similar that make it clear that the result is a string and that no one should make logic out of it.

Reply:
Makes sense, will do!